libs/api-client-tiger/src/client.ts

```diff
@@ -43,14 +45,23 @@ export {
 
 export interface ITigerClient {
     axios: AxiosInstance;
-    workspaceObjects: ReturnType<typeof tigerWorkspaceObjectsClientFactory>;
     execution: ReturnType<typeof tigerExecutionClientFactory>;
     executionResult: ReturnType<typeof tigerExecutionResultClientFactory>;
     labelElements: ReturnType<typeof tigerLabelElementsClientFactory>;
     validObjects: ReturnType<typeof tigerValidObjectsClientFactory>;
-    organizationObjects: ReturnType<typeof tigerOrganizationObjectsClientFactory>;
     explain: ReturnType<typeof tigerAfmExplainClientFactory>;
     declarativeLayout: ReturnType<typeof tigerLayoutClientFactory>;
+    entities: ReturnType<typeof tigerEntitiesObjectsClientFactory>;
+
+    /**
+     * @deprecated use entities {@link entities} instead
```

Comment:
Should most likely be `{@link ITigerClient.entities}`. Is the api-extractor build passing?